tests/smoke-tests/test-block-finalized.ts

```diff
@@ -0,0 +1,90 @@
+import "@moonbeam-network/api-augment";
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+
+const debug = require("debug")("smoke:block-finalized");
+const wssUrl = process.env.WSS_URL || null;
+const relayWssUrl = process.env.RELAY_WSS_URL || null;
+const ethRpcUrl = process.env.ETH_URL || null;
+
+describeSmokeSuite(
+  `Parachain blocks should be finalized..`,
+  { wssUrl, relayWssUrl, ethRpcUrl },
+  (context) => {
+    it("should have a recently finalized block", async function () {
+      const head = await context.polkadotApi.rpc.chain.getFinalizedHead();
+      const block = (await context.polkadotApi.rpc.chain.getBlock(head)).toHuman() as any;
+      const diff = Date.now() - getBlockTime(block);
```

Comment:
While it is often good to separate functions. I suggest for tests, to contain most of the code inside the test itself (so readers don't have to jump around everywhere to follow what the test is doing).

However, this function `getBlockTime` seems generic and useful enough to include it in our tests/util/block.ts 
It needs to better support the types

Reply:
Done